main/src/com/google/refine/grel/ControlFunctionRegistry.java

```diff
@@ -236,11 +238,18 @@ static public void registerControl(String name, Control c) {
 
         // HTML functions from JSoup
         registerFunction("parseHtml", new ParseHtml());
-        registerFunction("select", new SelectHtml());
-        registerFunction("htmlAttr", new HtmlAttr());
-        registerFunction("htmlText", new HtmlText());
+        registerFunction("select", new SelectXml());
+        registerFunction("htmlAttr", new XmlAttr());
+        registerFunction("htmlText", new XmlText());
         registerFunction("innerHtml", new InnerHtml());
         registerFunction("ownText", new OwnText());
+        
+        // XML functions from JSoup
+        registerFunction("parseXml", new ParseXml());
+        registerFunction("selectx", new SelectXml());
```

Comment:
Why are we registering `SelectXml` both as `select` and `selectx`? Isn't `select` enough?

Reply:
Good catch 

That’s a mistake on my part - I originally added a new function then removed it (incompletely it turns out) when I realised it wasn’t needed.

I’ll fix it and update the PR